backend/src/main/java/com/huellapositiva/infrastructure/orm/repository/JpaVolunteerRepository.java

```diff
@@ -39,4 +40,7 @@
     @Transactional
     @Query("UPDATE JpaVolunteer p SET p.location = :location WHERE p.id = :id")
     Integer updateLocation(@Param("id") String id, @Param("location") JpaLocation location);
+
+    @Query("FROM JpaVolunteer v LEFT JOIN FETCH v.credential c WHERE v.profile.subscribed = true")
+    List<JpaVolunteer> findSubscribedVolunteers();
```

Comment:
For Hibernate is better to use the complete QUERY.

Reply:
Solved